tests/test_models.py

```diff
@@ -1,7 +1,9 @@
 """Tests for statistics functions within the Model layer."""
 
+import os
```

Comment:
We are not using os package in this testing_models for this file, does this import make sense here?